components/ui/stickerSelector.tsx

```diff
@@ -0,0 +1,41 @@
+import { useStickerStore } from "@/stores/sticker.store";
+import { useTranslations } from "next-intl";
+import Image from "next/image";
+import { stickerItems } from "./stickerItems";
+
+type Sticker = {
+  id: number;
+  path: string;
+  bgPath: string;
+};
+
+const StickerSelector = () => {
+  const t = useTranslations("StickerSelector");
+
+  const stickerId = useStickerStore(
+    (state: { stickerId: number }) => state.stickerId,
+  );
+  const updateSticker = useStickerStore(
+    (state: { updateSticker }) => state.updateSticker,
+  );
+  return (
+    <div className="z-2 absolute mt-[154px] flex h-[560px] w-[342px] flex-col items-center gap-6 rounded-xl bg-[#F8F8F7] py-6">
+      <p className="text-base font-semibold text-[#6C1F1F]">{t("sticker")}</p>
+      <div className="flex w-[253px] flex-wrap items-center gap-x-[50px] gap-y-4">
+        {stickerItems.map((item: Sticker) => (
```

Comment:
If stickerItems' type is explicitly declared, `.map` will automatically infer the type. It's a bad practice to override typing.

The point of typescript is to override type when necessary only and let type do the work for us, instead of us working to make the type work